packages/saltcorn-markup/table.ts

```diff
@@ -89,7 +91,10 @@ const mkTable = (
   opts: OptsParams | any = {}
 ): string =>
   div(
-    { class: "table-responsive" },
+    {
+      class: `table-responsive ${opts.tableClass ? opts.tableClass : ""}`,
```

Comment:
class can take an array which omits falsy value: `class: ["table-responsive", opts.tableClass]`

this would also not have the space at the end, so i think you can roll back some/all changes to exact_views